indico/modules/events/registration/controllers/__init__.py

```diff
@@ -41,14 +41,22 @@ def _get_file_data(self):
 
     def _get_optional_fields(self):
         """Get fields for which we already have a value and thus are not required."""
+        if self.management:
+            # In management mode, all fields are optional
+            return True
+
         data_by_field = self.registration.data_by_field
         return [form_item.html_field_name for form_item in self.regform.active_fields
                 if data_by_field.get(form_item.id) is not None]
 
     def _process_POST(self):
         optional_fields = self._get_optional_fields()
+        override_required = request.json.get('override_required', False)
```

Comment:
Do we need this for modifications? We only send the field when creating the registration

Reply:
Yes I think we can do without here.